ui/components/app/confirm-page-container/confirm-page-container.component.js

```diff
@@ -142,7 +142,7 @@ const ConfirmPageContainer = (props) => {
       fromAddress,
       global.ethereumProvider,
     );
-    setCollectionBalance(tokenBalance?.balance?.words?.[0] || 0);
+    setCollectionBalance(tokenBalance.toString(10) || '0');
```

Comment:
```suggestion
    setCollectionBalance(tokenBalance.toString() || '0');
```

The update to use the `toString` method lgtm! One thing is passing a parameter does through a warning. 
https://github.com/ethers-io/ethers.js/blob/master/packages/bignumber/src.ts/bignumber.ts#L201

> BigNumber.toString does not accept any parameters; base-10 is assumed

Reply:
@digiwand  Referring to your previous comment, will it be possible to verify this change without the base won't break in firefox? 
Firefox build is not working in my local, so I cannot test it. :( 

> decimals and balance are not converted from BNs to strings automagically in Firefox (as they are in Chrome) so we need to do this for now.
> 
> decimals: details?.decimals?.toString(10),
> balance: details?.balance?.toString(10),